x-pack/plugins/observability_onboarding/public/assets/standalone_agent_setup.sh

```diff
@@ -71,19 +72,18 @@ if [ "$?" -eq 0 ]; then
   echo "Downloaded Elastic Agent"
   updateStepProgress "ea-download" "complete"
 else
-  updateStepProgress "ea-download" "danger"
+  updateStepProgress "ea-download" "danger" "Failed to download Elastic Agent, see script output for error."
```

Comment:
Could it be possible here to send the actual error to kibana?

Reply:
depending on the error, it could be difficult to parse out an error message from the right output stream in the script. and i think folks are OK with a generic message here to be directed to the host for more details.